verify_phpunit_xml/verify_phpunit_xml.sh

```diff
@@ -65,6 +65,7 @@ unittestclasses="
     mod_assign\\\\externallib_advanced_testcase
     mod_lti_testcase
     mod_quiz_attempt_walkthrough_from_csv_testcase
```

Comment:
Should we remove this one as well?

Reply:
No. See commit messages 🤣
We still run this on older branches and those outside our control.